stream_alert/apps/_apps/salesforce.py

```diff
@@ -57,7 +57,7 @@ class SalesforceApp(AppIntegration):
     ReportExport:
         events contain details about reports that a user exported.
     """
-    _SALESFORCE_TOKEN_URL = 'https://login.salesforce.com/services/oauth2/token'
+    _SALESFORCE_TOKEN_URL = 'https://login.salesforce.com/services/oauth2/token' #nosec
```

Comment:
definitely a nitpick, but to be consistent with else where, can you update these comments to be `# nosec` (space after `#`)

Reply:
😨 ooops, I thought had to be one word. LOL. Will correct it after rebase this branch.